### ISSUE TITLE
plotProfile trims data when supplying `plim`

### ISSUE THREAD

**Issue by richardsc** [dankelley/oce#1137]
This is perhaps an aesthetic thing, but I really don't like the way that `plotProfile()` "trims" data when plotting with axis limits. Of course, this is an aesthetic issue only because of R's standard approach to plot limits that they actually extend a little beyond the actual supplied limits, e.g. for the `par(yaxs='r')` (which I think is something like 4% extension).
```r
library(oce)
data(ctd)
plotProfile(ctd, plim=c(20, 10))
```
![image](https://cloud.githubusercontent.com/assets/233584/21275463/53ff5120-c3a3-11e6-9263-a0c0d37bbc6e.png)

Presumably this is done to save time when plotting only a small subset of a long series, by only passing the data that fall within the plot limits, but my problem is that it makes it look like there are no data beyond the limits.

I'd propose that instead of trimming to the actual limits, the data should be trimmed to slightly more (say 5%) so that the continuation of data past the specified limit is obvious.

Thoughts, @dankelley ? I'd be happy to try and tackle this if you agree.

**Comment by dankelley** [comment]
I agree -- that looks bad. It is also at odds with normal R behaviour
```R
> plot(1:10,xlim=c(3,5),type='l')
> par('usr')
2.92  5.08  0.64 10.36
```
and so it should be changed.  Maybe you could give it a shot? We should mimic normal R behaviour which is, as the above shows, to extend the plot a bit.

**Comment by richardsc** [comment]
Assigning this to @dankelley -- I'm probably not going to get to it over the holiday (no rush, though).

**Comment by dankelley** [comment]
Fixed in `develop` commit 5fa4bb1379c96fbe6824d4313da67625298c4da5 e.g.
```R
library(oce)
data(ctd)
plotProfile(ctd, plim=c(20, 10))
```
makes as below. To avoid bugging Clark, I will close this now (on the understanding that he should feel free to reopen it, of course)
![1137](https://cloud.githubusercontent.com/assets/99469/21501280/ecabe6f0-cc1c-11e6-9507-699503b83532.png)
